service/collector/elb.go

```diff
@@ -120,16 +125,18 @@ func (e *ELB) collectForAccount(ch chan<- prometheus.Metric, awsClients clientaw
 		return microerror.Mask(err)
 	}
 
-	var loadBalancers []*elb.LoadBalancerDescription
+	var loadBalancerNames []*string
```

Comment:
Only `LoadBalancerName` is used so to make further logic in this function nicer, extract name out here.